com.eclipsesource.modelserver.client/src/main/java/com/eclipsesource/modelserver/client/ModelServerClient.java

```diff
@@ -291,33 +312,25 @@ public void onFailure(@NotNull WebSocket webSocket, @NotNull Throwable t, @Nulla
     public boolean unsubscribe(String modelUri) {
         final WebSocket webSocket = openSockets.get(modelUri);
         if (webSocket != null) {
-            final boolean closed = webSocket.close(1000, "Websocket closed by client.");
-            return closed;
+            return webSocket.close(1000, "Websocket closed by client.");
         }
         return false;
     }
 
 
-    private String makeUrl(String path) {
-        return baseUrl + path;
-    }
-
     private String makeWsUrl(String path) {
         return (baseUrl + path).replaceFirst("^http(s?):", "^ws$1:");
```

Comment:
We could use the `makeUrl()` method here, and then take advantage of an eventual enhancement in it to ensure exactly one slash between the `baseUrl` and the `path` ;-)